ios/ElectrodeApiImpl/APIImpls/Navigation/ENNavigationDelegate.swift

```diff
@@ -228,12 +228,14 @@ import UIKit
     @objc func clickButtonWithButtonId(_ sender: ENBarButtonItem) {
         if let stringTag = sender.stringTag {
             ENNavigationAPIImpl.shared.navigationAPI.events.emitEventOnNavButtonClick(buttonId: stringTag)
+            ENNavigationAPIImpl.shared.navigationAPI.events.emitEventNavEvent(eventData: NavEventData(eventType: NavEventType.BUTTON_CLICK.rawValue, viewId: nil, jsonPayload: getButtonJsonPayload(buttonId: stringTag)))
```

Comment:
We need to pass a viewId for button clicks as well. The React Native component looks for this id to find the right view instance to fire the event in case the same component is mounted more than once.